### PR TITLE
Checked the astro configs for the templates,

### PR DESCRIPTION
**Pull request by aFuzzyBear**
## Changes

Minor changes, just went through the configs to check the renderers in each, tbh they were all fine, just a couple needing some touching up...
## Testing
Loaded up each templates, they all worked fine, esp the `framework-multiple`

### CODE REVIEW

File: examples/minimal/astro.config.mjs [hunk 1]
```diff
@@ -9,5 +9,5 @@
 // @ts-check
 export default /** @type {import('astro').AstroUserConfig} */ ({
   // Set "renderers" to "[]" to disable all default, builtin component support.
-  renderers: [],
+  // renderers: [],
```
Comment (aFuzzyBear) [inline examples/minimal/astro.config.mjs:12]: Muted these

Comment (FredKSchott) [reply examples/minimal/astro.config.mjs]: this one is actually intentional! I just push a commit to rewrite to make this more clear

**Review by FredKSchott — APPROVED** [review]
LGTM!

File: examples/minimal/astro.config.mjs (hunk 1 above)
Comment (aFuzzyBear) [reply examples/minimal/astro.config.mjs]: ah, i see, sorry for that man

**Pull request merged**